src/examples/forms/input-group/Readonly.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Field, Label, InputGroup, Input } from '@zendeskgarden/react-forms';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row justifyContent="center">
+    <Col sm={5}>
+      <Field>
+        <Label>Plant</Label>
+        <InputGroup>
+          <Input value="Soil" readOnly />
+          <Button focusInset>Copy</Button>
+        </InputGroup>
```

Comment:
@m-lai this is probably the only example where we want to demonstrate "Copy". But let's think of something inside our world of "nursery/farming/etc" that could actually be copied (idk, a floral invoice?)

Reply:
@jzempel  @jzempel What about [this example for read-only](https://share.getcloudapp.com/4guJe48Q) where we use "Copy"?

I redid the whole bunch of Input Group examples to see if they could all fit better with the Garden theme and make more sense - thoughts?

https://share.getcloudapp.com/o0umX9L5 cc @austingreendev

Reply:
I updated the individual images in the [Input Group documentation](https://docs.google.com/document/d/1rf0NnH38fcVNoa2lFbQdlienJbj17t70HU3NPnqaYa8/edit#) if it helps add clarity, esp to the Placement example with the menu

Reply:
@m-lai these are feeling quite a bit nicer.

Reply:
@jzempel Thanks, I'm glad you brought it up! V has added some edits to the Google Doc itself, and I [updated the images for Button Placement ](https://share.getcloudapp.com/NQu14Wwj)so that there's one button instead of two. But otherwise, I think those are the only major changes since yesterday.

Reply:
Updated with the new copy and examples.